cds-snc/platform-forms-client
# Fix/a11y misc

# Summary | Résumé

The changes include some minor accessibility fixes:
 - Fixes Verify "pages" not updating the HEAD title
 - Fixes Verify "pages" not focussing the page header on load
 - Fixes lost focus when going from Verify to ReVerify
 - Fixes Form Builder Share dropdown sub menu to be activate-able by keyboard when a form has no title

Test the Verify page changes:
- start VoiceOver (or similar)
- navigating to the Sign In page, Sign In
- when the Verify "page" loads, the header should be focussed and announced, the page title updated as well
- click the "Send a new code" link
- when the Re-Send code "page" loads, the header should be focussed and announced, the page title updated as well
- click the "re-send code" button
- when the Verify "page" loads, the header should be focussed and announced, the page title updated as well

Test the Share dropdown sub menu:
- navigate to Form-builder, Design a form
- using the keyboard tab to the Share menu in the header and key enter
- the sub menu should open, key enter again
- the Form title field should be focussed 
- (bonus: do the above running VoiceOver and when the sub menu is focussed the "Activate to move.." text should be announced alongside the visible text)
